src/NoFrixion.MoneyMoov/Models/Payouts/Payout.cs

```diff
@@ -342,10 +343,16 @@ public Counterparty? DestinationAccount
     public bool HasCurrentUserAuthorised {  get; set; }
 
     /// <summary>
-    /// A list of the email addresses of all the users who have usccessfully authorised the latest version of the payout.
+    /// A list of the email addresses of all the users who have successfully authorised the latest version of the payout.
     /// </summary>
+    [Obsolete("Refer to Authorisations instead.")]
     public List<string>? AuthorisedBy { get; set; }
 
+    /// <summary>
+    /// A list of the users who have successfully authorised the latest version of the payout and when.
+    /// </summary>
+    public List<Authorisation>? Authorisations { get; set; }

```

Comment:
`CanBeNull` ?

Reply:
`[CanBeNull]` seems to be a JetBrains annotation, which isn’t widely used in this project. Adding the `?` already handles the nullable part.

Reply:
Yeah, it just seemed weird to me that this prop on Beneficiary.cs has it while these don't.

Reply:
Yep, I’ll leave it as it is in Beneficiary.cs since that’s how it was before. I tried removing it and getting rid of #nullable disable, but that opened a big can of worms requiring changes to multiple files in the API to properly handle null checks. So, I’ll leave it as is for now.